File: rpg-client-java/src/niveles/Nivel4.java
package niveles;

import utilidades.Acciones;
import utilidades.Constants;

import static utilidades.Acciones.*;

public class Nivel4 {

	private static Acciones acciones;

	public static void iniciar() {

		acciones = new Acciones();
		acciones.start();

		reset();
		aparecerCultivo(Constants.HUERTO_LUGAR_H1, Constants.HUERTO_CULTIVO_LECHUGA);
	}

	public static void finalizar() {
		end();
	}
}

File: rpg-client-python/src/niveles/nivel4.py
from src.utilidades.constants import CONSTANTS
from src.utilidades.acciones import *

def iniciar():
    reset()
    aparecerar_cultivo(
        CONSTANTS.HUERTO_LUGAR_H1,
        CONSTANTS.HUERTO_CULTIVO_LECHUGA)

def finalizar():
    end()

